onnxruntime/core/providers/webnn/builders/impl/gemm_op_builder.cc

```diff
@@ -30,6 +30,72 @@ class GemmOpBuilder : public BaseOpBuilder {
                                const logging::Logger& logger) const override;
 };
 
+// Helper function
+common::Status ProcessZeroPointAndScale(
+    const std::vector<int64_t>& input_shape,
```

Comment:
```suggestion
    gsl::span<const int64_t> input_shape,
```
Postel's law, be permissive in what you accept and explicit in what you return.

Reply:
Done.